### PR TITLE
[Processing] Database algorithms chapter

### PR DESCRIPTION
**Pull request by ghtmtt**
With this small PR I would like to start the Processing algorithm description updates:
For instance, this PR:

* updates the text of the algorithms
* adds (only for PostGIS for the moment) also the new usage from console
* no images have been added

As discussed in the mailing list, it is better to have the updated description already upstream than keeping it "at home" ;)

### CODE REVIEW

**Comment by ghtmtt** [comment]
incorporates also the changes of #1969

File: source/docs/user_manual/processing_algs/qgis/database.rst [hunk 1]
```diff
@@ -59,51 +56,215 @@ Parameters
...
-
 Allows a SQL database query to be performed on a PostGIS database connected to QGIS.
+The algorithm **won't** create any new layer: it is designed to run queries on
+the layer itself.
+
+.. _qgis_postgis_execute_sql_example:
+
+Example
+.......
+1. Set all the values of an existing filed to a fixed value. The SQL query string
```
Comment (yjacolin) [inline source/docs/user_manual/processing_algs/qgis/database.rst]: filed -> field

File: source/docs/user_manual/processing_algs/qgis/database.rst [hunk 2]
```diff
@@ -59,51 +56,215 @@ Parameters
...
+will be::
```
Comment (yjacolin) [inline source/docs/user_manual/processing_algs/qgis/database.rst]: I think we should use `.. code-block:: sql` (or Python instead of sql) instead of `::`

**Comment by ghtmtt** [comment]
@yjacolin can I merge it?

**Comment by ghtmtt** [comment]
@DelazJ if you are not against it, I'd like to merge this PR.

File: source/docs/user_manual/processing_algs/qgis/database.rst [hunk 3]
```diff
@@ -59,51 +56,221 @@ Parameters
...
+No new outputs will be created. The layer chosen will be updated with the executed
+SQL query. By opening the table (for example with Data Manager) you will see
+the results.
+
+Console usage
+.............
+
+.. code-block:: python
+
+  # import processing
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/database.rst]: ditto

File: source/docs/user_manual/processing_algs/qgis/database.rst [hunk 4]
```diff
@@ -59,51 +56,221 @@ Parameters
...
+
 
 Console usage
 .............
 
-::
+.. code-block:: python
 
-  processing.runalg('qgis:importintopostgis', input, database, schema, tablename, primary_key, geometry_column, overwrite, createindex, lowercase_names, drop_string_length)
+  # import processing
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/qgis/database.rst]: import processing class

**Comment by ghtmtt** [comment]
@DelazJ no hurry, thanks for the feedbacks. I had to merge instead of rebasing (some conflicts). I'll wait Travis and the squash the PR. 

Thanks again!

**Pull request merged**